buildsnippets/snippets.go

```diff
@@ -97,11 +97,13 @@ var snippets = map[string]Snippet{
 
 func load(loc string) []string {
 	loc = filepath.Join("rawsnippets", loc)
-	f, err := os.Open(loc)
+	f, err := os.Open(filepath.Clean(loc))
 	if err != nil {
 		panic(err)
 	}
-	defer f.Close()
```

Comment:
While I understand the reasoning behind the change, this is the standard convention, and it performs exactly the same as your proposed change. This convention is described in the [Effective Go guide](https://golang.org/doc/effective_go#defer), which is referred to when styling conventions are called into question.

> This document gives tips for writing clear, idiomatic Go code. It augments the language specification, the Tour of Go, and How to Write Go Code, all of which you should read first.

I would suggest that whichever tool you're using to identify this issue should b updated to follow this convention.

Reply:
@bacongobbler thanks for your feedback
Salus uses gosec under the hood, and I'm quite sure they might be not very cooperative regarding this rule.
at least their recommendation https://github.com/securego/gosec/issues/512 here proposes to explicitly handle errors.

will silencing this particular warning be considered?